services/brig/docs/swagger.md

```diff
@@ -1,4 +1,17 @@
-## General
+## Authentication / Authorization
+
+The end-points in this API support differing authorization protocols:
+some are unauthenticated (`/api-version`, `/login`), some require
+[zauth](), and some support both [zauth]() and [oauth]().
+
+The end-points that require zauth are labelled so in the description
+below.  The end-points that support oauth as an alternative to zauth
+have the required oauth scopes listed in the same description.
+
+Futher reading:
+- https://docs.wire.com/how-to/install/oauth.html
+- https://github.com/wireapp/wire-server/blob/develop/libs/wire-api/src/Wire/API/Routes/Public.hs (search for HasSwagger instances)
+- `curl https://staging-nginz-https.zinfra.io/v4/api/swagger.json | jq '.security, .securityDefinitions`
```

Comment:
this is not up to date any more; we don't have swagger docs about oauth, nor `HasSwagger` instances.  what we can do (at least in this PR) is mark an end-point as "can either do zauth, or oauth, or both".

in the long run, we may want to read the nginz config into a small TH blob that renders the information into swagger from there?  doesn't strike me as particularly hard or hard to maintain.